src/client-websocket.ts

```diff
@@ -0,0 +1,11 @@
+declare let window: any;
+let _global = typeof global !== 'undefined' ? global : (typeof window !== 'undefined' ? window: {});
+
+import * as NodeSocket from 'ws';
```

Comment:
I changed the implementation of the client package, and now the client package look for the native `WebSocket` object, and you can provide an optional implementation if you are using Node. 
This way, `ws` package is not part of the Meteor client bundle, and not loaded at all.

Other option, is to use object for `browser` field and specify that `ws` is ignored for browsers. But this is not supported yet for Meteor build (https://github.com/meteor/meteor/issues/6890).